frameit/README.md

```diff
@@ -36,6 +36,8 @@ frameit
 
 ###### Quickly put your screenshots into the right device frames
 
+`frameit` allows you to put a gorgeous device frame around your iOS screenshots just by running one simple command. It’s perfect to generate screenshots for the App Store, your website, QA or emails. 
+
```

Comment:
Second sentence should be something to the tune of "Use `frameit` to prepare your screenshots for the App Store..." 

`snapshot` generates the screenshots, `frameit` prepares them.